esque/resources/topic.py

```diff
@@ -43,7 +42,7 @@ def as_dict(self):
             "partition_isrs": self.partition_isrs,
             "partition_leader": self.partition_leader,
             "partition_replicas": self.partition_replicas,
-            "latest_message_timestamp": self.latest_message_timestamp,
+            "latest_message_timestamp": self.latest_message_timestamp or "Missing due to timeout.",
```

Comment:
`self.last_message_timestamp` will also be `None` when `high_water == low_water`.
I'd just keep it as None. Other logic can get quite complicated.